### ISSUE TITLE
Display first day of the week in Reminders depending on the user settings (Linux)

### ISSUE THREAD

**Issue by skel35** [toggl-open-source/toggldesktop#3993]
### 💻 Environment
Platform: Linux

### 📒 Description
Implement #3922 for Linux.
Use `toggl_get_user_beginning_of_week` API method, added during implementation of #3922.

**Comment by MartinBriza** [comment]
This feature depends on `feature/autotracker-update-mvp` being merged into `master`.
Work in progress PR is now in the `feature/linux-day-of-week-order` branch (able to handle the changes but not actually getting any from the library)

**Comment by skel35** [comment]
I'll assign myself here, for now, as a reminder that I should rebase the `feature/autotracker-update-mvp` branch and merge to `master` whatever is possible.

**Comment by skel35** [comment]
unblocked
